next.config.ts

```diff
@@ -20,8 +20,16 @@ const getServerRuntimeConfig = (phase: string) => {
 
 const nextConfig: NextConfig = (phase: string) => {
     return {
+        // Disable compression to avoid memory issues
+        compress: false,
```

Comment:
How much does this increase the size of RRE page loads? Is it tenable?

Reply:
When I was testing earlier it was measuring in kbs, so I suspect minimal